### DISCUSSION TITLE
Degré de priorité

### DISCUSSION THREAD

**Discussion by tbroyer** [cnumr/best-practices#197] [Q&A] [answered]
@ACTLEM Je ne comprends pas la formule de calcul du degré de priorité: `ARRONDI.SUP(((MEO*IMPACT)/25)*5;0)` que tu mets dans les issues (et qu'on peut d'ailleurs simplifier en `ARRONDI.SUP((MEO*IMPACT)/5;0)`)

Une BP difficile à mettre en œuvre (MOE=1) mais à fort impact (5) aura une priorité de 1 :thinking: 

Pour la BP#6 on a en V3 MOE=difficile, impact=fort et priorité=conseillé, donc je m'attendrais plutôt à une priorité de 3 que de 1 si je devais la "mapper" plutôt que la calculer avec cette formule.

**Comment by ACTLEM** [comment]
Je suis d'acord @tbroyer. J'avoue que je n'ai pas encore pris le temps de challenger cette formule. Il faudrait demander à @bordage le sens de la formule.
Personnellement, je prendrais la moyenne et 3 possibilités en cas de 1/2 valeur:
- Arrondi sup pour favoriser vers le haut
- Arrondi inf vers le bas
- Priorité sur la MOE ou les impacts (prendre la valeur la plus proche du critère choisi)

Perso, je prendrais la 3ème solution avec les impacts en critère "supérieur".
Après, si l'on arrive à faire un sondage/vote avec avec beaucoup de contributeurs, on devrait avoir des moyennes généralement ne tombant pas sur du 1/2.

**Reply by Trankilloman** [reply]
En fait, la question est de savoir si le guide favorise :
- les solutions faciles à mettre en place avec un faible potentiel de réduction d'impacts (mais une réduction quand même)
- les solutions plus complexes a mettre en place mais avec un fort potentiel de réduction d'impacts (mais avec un risque de ne jamais pouvoir mettre en place ses solutions de réduction d'impacts).

C'est toujours un choix difficile qui dépend de la stratégie de la structure qui utilise les bonnes pratiques ou ici, de la stratégie du guide.
Personnellement (et celles et ceux que je connais du projet ecometer sont bien au courant ;-) ) je serais partisans de ne pas prioriser néanmoins, je pense qu'ici il ne serait pas déconnant d'avoir 2 classement : un pour les plus faciles et l'autre pour les plus efficiantes.

**Reply by ACTLEM** [reply]
Je ne suis pas loin de te rejoindre sur la suppression de la priorité ^^ (les 2 classements se faisant que sur l'un des deux critères).
Après, cela reste un guide qui est pris par certaines personnes comme un premier pied à l'étrier et avoir une priorisation peut les aider dans la prise de décision dans un premier temps. Une fois plus à l'aise avec la démarche ils pourront prendre plus de recul sur la priorité et prioriser par rapport à leur sensibilité (celles plus complexes mais avec le plus d'effet ou celles plus faciles et donc plus nombreuses mais avec moins d'impact unitairement).

**Comment by bordage** [comment, marked as answer]
Bonjour à tous,

1. Ordre des priorités
L'ordre de priorité est 1 = prioritaire et 3 = non prioritaire car ce n'est pas un poids mais une priorité (qu'est ce que je mets en oeuvre ne premier). On peut évidemment inverser. Ce n'est qu'une convention.

2. Nombre de niveaux
Certaines bonnes pratiques sont incontournables (par exemple BP1 et BP2) car elles ont un impact énorme. Afin de mieux refléter cet effet de levier, je préconise de passer sur une échelle de 5 au lieu de 3. Cela permettra de mieux faire ressortir les BP incontournables.

3. Calcul de la priorité
On peut imaginer ajouter un critère qualitatif "incontournable" et avoir une formule du type : 
( (incontournable * 3 + impact * 2 + MEO * 1) / 6 ) * 5

4. Objectif
La priorité de MEO vise à pointer du doigt les BP à priori les plus efficaces ET à simplifier les vie des néophytes qui débarquent sur le sujet. 

5. Décision
Dans un souci de continuité par rapport à l'existant et pour aider les néophytes, il me semble crucial de conserver ce niveau de priorité qui est très apprécié par les équipes sur le terrain. C'est vraiment un guide utile lorsqu'on est en mode projet pour éviter de rentrer dans un effort marginal. Par ailleurs, notre système d'évaluation de maturité repose en partie sur cette pondération.
